src/main/groovy/life/qbic/oncostore/model/VcfInfo.groovy

```diff
@@ -119,4 +122,16 @@ class VcfInfo {
     Boolean getValidated() {
         return validated
     }
+
+    String toVcfFormat() {
+        def vcfInfoString = new StringJoiner(VcfConstants.PROPERTY_DELIMITER)
+        this.properties.each { it ->
+            if (it.key != "class" & it.value != null) {
+                def name = it.key.toString().toUpperCase() as VcfConstants.VcfInfoAbbreviations
+                vcfInfoString.add("${name.getTag()}${VcfConstants.PROPERTY_DEFINITION_STRING}${it.value.toString()}")
+            }
+        }
```

Comment:
I would suggest some form of Exception Handling here concerning vcfInfoString to inform the user if a property is missing.

Reply:
Do you mean if the value is `null` for any of the defined properties?

Reply:
Yes, although i don't know if that case could occur. 
Right now as a User i wouldn't know the reason why the else case occurs(was it.key == "class" or was it's value == null?). 
I'd suggest a notification that states the property values explicitly in the else case.

Reply:
The reason for that is that most of these properties are optional in `VCF` files. Most of them will not be there in most cases. If they were there in the imported `VCF` file, they will be there in the exported one as well...

Reply:
Ah, Thanks for clarifying! Then I don't think it matters since the user can check the exported VCF file